frontend/src/components/PostItem.tsx

```diff
@@ -22,6 +22,10 @@ export default function PostItem(props: Props) {
     }, [router, link]);
     
     const stopPropagation = useCallback(e => e.stopPropagation(), []);
+
+    if (!post.banner) {
+        post.banner="../../images/banner.webp"
```

Comment:
Zou mooi zijn als deze instelbaar wordt vanuit de backend. Hiervoor moeten we een nieuwe single-type collection aanmaken waarin je de default post banner zou kunnen instellen. Op alle pagina's waar PostItems / Posts getoond worden, wil je dan het URL naar deze banner ophalen zodat je het kan gebruiken.

Reply:
> Zou mooi zijn als deze instelbaar wordt vanuit de backend. Hiervoor moeten we een nieuwe single-type collection aanmaken waarin je de default post banner zou kunnen instellen. Op alle pagina's waar PostItems / Posts getoond worden, wil je dan het URL naar deze banner ophalen zodat je het kan gebruiken.

Yeah dit zou beter zijn inderdaad. Ik had de functie `fetchFallbackBanner` gevonden ergens, maar die deed niet wat ik verwachte dat ie deed (of ik snap niet hoe ik die moet gebruiken).

Reply:
> Zou mooi zijn als deze instelbaar wordt vanuit de backend. Hiervoor moeten we een nieuwe single-type collection aanmaken waarin je de default post banner zou kunnen instellen. Op alle pagina's waar PostItems / Posts getoond worden, wil je dan het URL naar deze banner ophalen zodat je het kan gebruiken.

Ik heb iets gedaan in deze richting, maar mijn plak en knipsel werkt niet en ik snap hier niet genoeg van om het op te lossen:
```
error - src/components/PostItem.tsx (30:40) @ PostItem
TypeError: Cannot read properties of undefined (reading 'banner')
28 |
29 |     if (!post.banner) {
> 30 |         post.banner=props.defaultbanner.banner
|                                        ^
31 |     }
32 |
33 |     return <div onClick={handleClick}
```